src/core/connection.c

```diff
@@ -6623,6 +6623,27 @@ QuicConnParamSet(
         return QUIC_STATUS_SUCCESS;
     }
 
+    case QUIC_PARAM_CONN_DSCP:
+    {
+        if (BufferLength != sizeof(uint8_t) || Buffer == NULL) {
+            Status = QUIC_STATUS_INVALID_PARAMETER;
+            break;
+        }
+
+        uint8_t DSCP = 0;
+        CxPlatCopyMemory(&DSCP, Buffer, BufferLength);
```

Comment:
Simplify?
```suggestion
        uint8_t DSCP = *Buffer;
```

Reply:
Won't the simplification cause a crash if the buffer is unaligned on ARM?